lib/features/tree/display.dart

```diff
@@ -105,12 +105,14 @@ class TreeDisplayState extends ConsumerState<TreeDisplay> {
         ? image = '$tempDir/model_tree_rpart.svg'
         : image = '$tempDir/model_tree_ctree.svg';
 
-    pages.add(
-      ImagePage(
-        title: 'TREE',
-        path: image,
-      ),
-    );
+    if (content.isNotEmpty) {
```

Comment:
How does this make sense? We have a general problem across the app with managing the image files and how to know they have been properly updated. I don't think this solves the issue? Explain how this fixes that issue?